typescript/vrotsc/src/compiler/transformer/helpers/actionTransformHelper.ts

```diff
@@ -0,0 +1,121 @@
+/*-
```

Comment:
Helpers don't do visits or modifications, used for easier access and such. Transformation should be coupled with the code that needs it (action.ts)

Code transformers do, but please don't move this there, only if it's reused and this won't be